### DISCUSSION TITLE
New Relic Workflow: mute issues

### DISCUSSION THREAD

**Discussion by Daemonali** [newrelic/terraform-provider-newrelic#2050] [Q&A]
Hey, 

I am trying to mimic the following mute issue in terraform via new relic workflows: `Do not send notifications for fully muted issues`

It is currently set to the default - `muting_rules_handling = "NOTIFY_ALL_ISSUES"` 

How would I change it so that it matches `Do not send notifications for fully muted issues` ? 

Many thanks

**Comment by IliaShurygin** [comment]
Hi! Sorry for the late reply! We have since updated [our TF docs](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/resources/workflow) to include the possible values for `muting_rules_handling` field.

Answering your specific question, you need to use `DONT_NOTIFY_FULLY_MUTED_ISSUES`
